nixos/release.nix

```diff
@@ -150,13 +150,35 @@ in rec {
   });
 
   iso_plasma5 = forMatchingSystems [ "x86_64-linux" ] (system: makeIso {
-    module = ./modules/installer/cd-dvd/installation-cd-graphical-plasma5.nix;
+    module = ./modules/installer/cd-dvd/installation-cd-graphical-calamares-plasma5.nix;
     type = "plasma5";
     inherit system;
   });
 
+  calamaresInstaller = with import ./.. { system = "x86_64-linux"; };
+    import lib/eval-config.nix {
+      inherit system;
+      modules = [
+        configuration
+        versionModule
+        ./modules/virtualisation/qemu-vm.nix
+        ./modules/installer/cd-dvd/installation-cd-graphical-calamares-gnome.nix
+        {
+          documentation.enable = mkVMOverride false;
+          documentation.nixos.enable = mkVMOverride false;
+          environment.defaultPackages = mkVMOverride [ ];
+
+          virtualisation = {
+            qemu.options = [ "-vga virtio" ];
+            memorySize = 2048;
+            emptyDiskImages = [ (8 * 1024) ];
+          };
```

Comment:
Would be nice to add comments explaining these.

Reply:
@bobvanderlinden I know you added this in https://github.com/bobvanderlinden/nixpkgs/commit/8c22ca1f081ccd2e9fca0c06d937ea929b4378cc, do you think this is still useful to have or unnecessary?

Reply:
Removed for now, could add back later as an enhancement